jp/components/geo-map-type-scatter-symbol-series.md

```diff
@@ -27,7 +27,7 @@ Angular マップ コンポーネントでは、[`IgxGeographicSymbolSeriesCompo
 
 ## コード スニペット
 
-以下のコードは、[`IgxShapeDataSource`]({environment:dvApiBaseUrl}/products/ignite-ui-angular/api/docs/typescript/latest/classes/igniteui_angular_core.igxshapedatasource.html) を使用してシェイプ ファイルからロードした都市の場所に [`IgxGeographicSymbolSeriesComponent`]({environment:dvApiBaseUrl}/products/ignite-ui-angular/api/docs/typescript/latest/classes/igniteui_angular_maps.igxgeographicsymbolseriescomponent.html) をバインドする方法を示します。
```

Comment:
These links are still coming out as `undefined`. I suppose this should be resolved.

Reply:
I fixed those undefined links